### DISCUSSION TITLE
Didn`t get the reawrd till now of bounty issue#2481

### DISCUSSION THREAD

**Discussion by HarjobandeepSingh** [osmosis-labs/osmosis-frontend#2514] [General]
I already shared my all details by email and at pull request but still didn`t recieve my reward #2505

**Comment by HarjobandeepSingh** [comment]
@jonator

**Comment by jonator** [comment]
I'd expect it to take some time cc @MaxMillington

**Reply by HarjobandeepSingh** [reply]
Hoping 🤞 i will get it soon 
![qr-image-1701971708370.png](https://github.com/osmosis-labs/osmosis-frontend/assets/81474463/fba70c40-9515-4887-b0ad-002ac96cae60)
